compiler/src/iree/compiler/Codegen/Utils/CPUUtils.cpp

```diff
@@ -63,4 +66,18 @@ FailureOr<Operation *> getRootOperation(ArrayRef<Operation *> computeOps) {
   return rootOperation;
 }
 
+StringAttr getEnableLoopPeelingAttrName(MLIRContext *ctx) {
+  return StringAttr::get(ctx, kLoopPeelingAttrName);
+}
+
+bool isLoopPeelingEnabled(FunctionOpInterface funcOp) {
+  DictionaryAttr config = getTranslationInfo(funcOp).getConfiguration();
+
+  if (!config) {
+    return false;
+  }
+
+  return config.contains(kLoopPeelingAttrName);
```

Comment:
nit: I think we can collapse it to `return config && config.contains(...);`. What do you think?

Reply:
Ben suggested exactly that and in my had I've already added that 🤦🏻 But that was imaginary code :) Thanks for pointing this out, sending update shortly.